purchase_date_planned_manual/models/purchase_order.py

```diff
@@ -52,3 +52,24 @@ def action_delayed_line(self):
             'This line is scheduled for: %s. \n However it is now planned to '
             'arrive late.') % Dt.to_string(Dt.context_timestamp(
                 self, Dt.from_string(self.date_planned))))
+
+    def _merge_in_existing_line(self, product_id, product_qty, product_uom,
+                                location_id, name, origin, values):
+        if self.date_planned == values.get('date_planned'):
+            return super(PurchaseOrderLine, self)._merge_in_existing_line(
+                product_id, product_qty, product_uom,
+                location_id, name, origin, values)
+        else:
```

Comment:
@mreficent here `else` is needed ??

I think It Should be a better :smiley: 
```
if self.date_planned == values.get('date_planned'):
    return super(PurchaseOrderLine, self)._merge_in_existing_line(
        product_id, product_qty, product_uom,
        location_id, name, origin, values)
return False
```

Reply:
done